src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.tizen.cs

```diff
@@ -0,0 +1,144 @@
+
+	/// <inheritdoc/>
+	protected override void Dispose(bool isDisposing)
+	{
+		if (isDisposing)
+		{
+			OutsideClicked -= OnOutsideClicked;
+		}
+
+		base.Dispose(isDisposing);
+	}
+
+	/// <summary>
+	/// Method to initialize the native implementation.
+	/// </summary>
+	/// <param name="element">An instance of <see cref="IPopup"/>.</param>
+	public void SetElement(IPopup? element)
+	{
+		VirtualView = element;
+	}
+
+	/// <summary>
+	/// Method to show the Popup
+	/// </summary>
+	public void ShowPopup()
+	{
+		_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} cannot be null");
+		Content = VirtualView.Content?.ToPlatform(mauiContext) ?? throw new InvalidOperationException($"{nameof(VirtualView.Content)} cannot be null");
+
+		if (Content is null)
+		{
+			return;
+		}
+
+		BackgroundColor = new Tizen.NUI.Color(0.1f, 0.1f, 0.1f, 0.5f);
+		Content.BackgroundColor = (VirtualView.Color ?? Colors.Transparent).ToNUIColor();
+
+		if (VirtualView.Anchor is not null)
+		{
+			var anchorView = VirtualView.Anchor.ToPlatform();
+			var anchorPosition = anchorView.ScreenPosition;
+			Content.ScreenToLocal(out float x, out float y, anchorPosition.X, anchorPosition.Y);
+			Layout = new AbsoluteLayout();
+			Content.UpdatePosition(new Tizen.UIExtensions.Common.Point(x, y));
+		}
+		else
+		{
+			Layout = new LinearLayout
+			{
+				LinearOrientation = LinearLayout.Orientation.Vertical,
+				VerticalAlignment = ToVerticalAlignment(VirtualView.VerticalOptions),
+				HorizontalAlignment = ToHorizontalAlignment(VirtualView.HorizontalOptions),
+			};
+			Content.UpdatePosition(new Tizen.UIExtensions.Common.Point(0, 0));
+		}
+
+		UpdateContentSize();
+
+		Open();
+		VirtualView.OnOpened();
+	}
+
+	/// <summary>
+	/// Method to update size of Content
+	/// </summary>
+	/// <exception cref="InvalidOperationException"></exception>
+	public void UpdateContentSize()
+	{
+		_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} cannot be null");
+		if (Content is null)
+		{
+			return;
+		}
+
+		if (VirtualView.Size.Width > 0 && VirtualView.Size.Height > 0)
+		{
+			Content.UpdateSize(VirtualView.Size.ToPixel());
+		}
+		else
+		{
+			var measured = VirtualView.Content?.Measure(double.PositiveInfinity, double.PositiveInfinity).ToPixel() ?? new Tizen.UIExtensions.Common.Size(0, 0);
+			Content.UpdateSize(measured);
+		}
+	}
+
+	static NVerticalAlignment ToVerticalAlignment(LayoutAlignment align) => align switch
+	{
+		LayoutAlignment.Start => NVerticalAlignment.Top,
+		LayoutAlignment.End => NVerticalAlignment.Bottom,
+		_ => NVerticalAlignment.Center
+	};
+
+	static NHorizontalAlignment ToHorizontalAlignment(LayoutAlignment align) => align switch
+	{
+		LayoutAlignment.Start => NHorizontalAlignment.Begin,
+		LayoutAlignment.End => NHorizontalAlignment.End,
+		_ => NHorizontalAlignment.Center
+	};
+
+	void OnOutsideClicked(object? sender, EventArgs e)
+	{
+		if (VirtualView is null || VirtualView.Handler is null)
```

Comment:
Can we simplify this?
```suggestion
		if (VirtualView?.Handler is null)
```

Reply:
Done